x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/entity_store_data_client.ts

```diff
@@ -137,7 +180,16 @@ export class EntityStoreDataClient {
 
     const { config } = this.options;
 
-    await this.riskScoreDataClient.createRiskScoreLatestIndex();
+    await this.riskScoreDataClient.createRiskScoreLatestIndex().catch((e) => {
```

Comment:
I think this should be handled inside `createRiskScoreLatestIndex` what do you think?

Reply:
Not sure...

It does fit with our overall approach but I'm worried it might be too strict of an abstraction. For example, if we ever get some feature where it is important to handle these errors (maybe we want to show something in the UI) we would be stuck 🤔 

In general I think it's probably best to leave each client/consumer handle the errors. Since this is such a common way for us to handle this particular error, maybe we can have some utility fn to reduce the boilerplate?